src/lavinmq/shovel/shovel.cr

```diff
@@ -44,7 +44,7 @@ module LavinMQ
                      @delete_after = DEFAULT_DELETE_AFTER, @prefetch = DEFAULT_PREFETCH,
                      @ack_mode = DEFAULT_ACK_MODE, consumer_args : Hash(String, JSON::Any)? = nil,
                      direct_user : User? = nil)
-        @tag = "Shovel[#{@name}]"
+        @tag = "Shovel-#{Random::Secure.urlsafe_base64(24)}"
```

Comment:
I don't know what this adds, afaik this is never logged anywhere. The name of the receiving queue would make more sense. Else, just `shovel`.

Reply:
Yeah, you're right. I was thinking we wanted to keep consumer tags unique, but I guess that doesn't matter here since they are always on their own channel.